gapic/templates/examples/sample.py.j2

```diff
@@ -1,18 +1,3 @@
-{#
- # Copyright (C) 2019  Google LLC
- #
- # Licensed under the Apache License, Version 2.0 (the "License");
- # you may not use this file except in compliance with the License.
- # You may obtain a copy of the License at
- #
- #     http://www.apache.org/licenses/LICENSE-2.0
- #
- # Unless required by applicable law or agreed to in writing, software
- # distributed under the License is distributed on an "AS IS" BASIS,
- # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- # See the License for the specific language governing permissions and
- # limitations under the License.
-#}
```

Comment:
Same here; I would have imagined we still need the copyright header.

Reply:
`_base.py.j2` now includes the license by default so it doesn't need to be duplicated here: https://github.com/googleapis/gapic-generator-python/blob/master/gapic/templates/_base.py.j2

Reply:
My concern is with the template itself, not the generated output. The template is also subject to copyright.

Reply:
Ah of course. Re-added.